cmd/config/internal/commands/cmdset.go

```diff
@@ -203,3 +232,23 @@ func (r *SetRunner) perform(c *cobra.Command, args []string) error {
 	fmt.Fprintf(c.OutOrStdout(), "set %d fields\n", r.Perform.Count)
 	return nil
 }
+
+// OpenAPIFilePaths returns the openAPI file paths in the root directory
+// returns only file path at top level if RecurseSubPackages is false
+// returns all the file paths in the nested subdirs if RecurseSubPackages is true
+func (r *SetRunner) OpenAPIFilePaths(root string) ([]string, error) {
+	if !r.Set.RecurseSubPackages {
+		if root == "." {
+			return []string{root + "/" + r.Set.OpenAPIFileName}, nil
```

Comment:
as done below, use `filepath.Join(root, r.Set.OpenAPIFileName)`

Reply:
i'm confused.

Reply:
filepath.join just returns the file name if the directory root is . in this case, but we want ./fileName as output. Anyways, I refactored this to make it more readable. PTAL again.